src/odemis/gui/util/__init__.py

```diff
@@ -115,12 +115,12 @@ def dead_object_wrapzor(*args, **kwargs):
             app = wx.GetApp()
             if app:
                 return f(*args, **kwargs)
-        except (wx.PyDeadObjectError, RuntimeError):
+        except RuntimeError:
             logging.warning("Dead object ignored in %s", f.__name__)
         except TypeError:
             # If a wx.Window is destroyed and then passed as a parameter, a TypeError might reulst
             # instead of a PyDeadObjectError.
-            if all([isinstance(a, wx._core._wxPyDeadObject) for a in args]):
```

Comment:
This is not the same. It used to check for objects which are dead, and now it check for object which are an `Exception`.
Probably the initial code is not correct, as it looks like it should use `any` instead of `all`. It also doesn't check in `kwargs`.
I don't know how to check for dead objects now, and I don't know if a `TypeError` would happen in such case anyway. So for now, I'd suggest to comment out this entire `except` block, and add a comment indicating why it is commented out.